### ISSUE TITLE
Issue with Object Placement on Face in tasks-vision.facelandmarker web

### ISSUE THREAD

**Issue by soroushmadani** [googlesamples/mediapipe#230]
I'm writing to report an issue I've encountered while working with the tasks-vision.facelandmarker in MediaPipe. I am trying to overlay virtual glasses on a user's face, and while everything works perfectly when the face is in the center of the camera view, I'm experiencing problems with the glasses' rotation when the face moves to the sides.

I suspect that the issue might be related to the facialTransformationMatrixes or some configuration settings, possibly with the perspective camera. To illustrate this problem, I've prepared a set of images that demonstrate the glasses' misalignment issue. You can view these images here: 
<img width="1279" alt="Screenshot 1402-06-17 at 12 58 20" src="https://github.com/googlesamples/mediapipe/assets/30050361/35927acc-6795-4f64-8371-1fddeda561f1">
<img width="1279" alt="Screenshot 1402-06-17 at 12 57 43" src="https://github.com/googlesamples/mediapipe/assets/30050361/7edf02c0-28cb-4d5e-a86c-d324e1f68f8d">

Additionally, I noticed that this problem also exists in your example on CodePen, which can be accessed via the following URL: https://codepen.io/mediapipe-preview/pen/oNPKmEy.
<img width="2237" alt="Screenshot 1402-06-17 at 12 55 04" src="https://github.com/googlesamples/mediapipe/assets/30050361/8faf6b2b-0d2a-4005-a587-0ff4a948a01c">
I wanted to note that I have made a modification to the scale of the avatar in the CodePen example, changing it from 40 to 20. This adjustment is intended to help clearly showcase the difference in angles between my face and the object.

Could you please provide guidance on how I can address this issue? I'm keen to ensure that the glasses' placement remains accurate, even when the user's face is not in the center of the camera view.

Thank you for your time and assistance in resolving this matter. I appreciate your dedication to the MediaPipe project and look forward to your guidance on resolving this issue.

**Comment by furkandindar** [comment]
does anyone has information/knowledge about this? I'm dealing with the same issue, couldn't figure out why it behaves like this

**Comment by Affan007** [comment]
yes , facing the same issue, @soroushmadani , how did you resolve this ?

**Comment by soroushmadani** [comment]
> yes , facing the same issue, @soroushmadani , how did you resolve this ?

I did not, keep me updated if you solved this, thank you.
